flagsmith-jira-app/src/IssueFlagPanel.tsx

```diff
@@ -186,25 +194,19 @@ const IssueFlagTable = ({
                       <Cell>
                         <Text>
                           <StatusLozenge
-                            appearance={flag.enabled ? "success" : "default"}
-                            text={flag.enabled ? "on" : "off"}
+                            appearance={environment.enabled ? "success" : "default"}
+                            text={environment.enabled ? "on" : "off"}
```

Comment:
I don't understand why we have `environment.enabled` here, shouldn't it be `environmentFlag.enabled` ?

Reply:
This was already changed in the latest version.